indico/modules/events/sessions/templates/management/_session_list.html

```diff
@@ -0,0 +1,109 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_session_list(event, sessions, default_colors) -%}
+    {% if sessions -%}
+        <table class="i-table tablesorter">
+            <thead>
+                <tr class="i-table">
+                    <th class="i-table checkbox-column" data-sorter="false"></th>
+                    <th class="i-table id-column">{% trans %}ID{% endtrans %}</th>
+                    <th class="i-table">{% trans %}Title{% endtrans %}</th>
+                    <th class="i-table">{% trans %}Code{% endtrans %}</th>
+                    <th class="i-table">{%- trans %}Type{% endtrans -%}</th>
+                    <th class="i-table" data-sorter="false">{% trans %}Colour{% endtrans %}</th>
+                    <th class="i-table" data-sorter="false">{% trans %}Blocks{% endtrans %}</th>
+                    <th data-sorter="false"></th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for sess in sessions -%}
+                    <tr class="i-table">
+                        <td class="i-table">
+                            <input class="select-row" type="checkbox" value="{{ sess.id }}"
+                                   name="session_ids">
```

Comment:
I would go for singular `session_id`. That's what we have in contributions and I think it makes more sense. `.getlist('session_id')` gets a list of `session_id` values